layouts/partials/nav.html

```diff
@@ -23,7 +23,14 @@
                   {{ $current := . }}
                   {{ range .Site.Menus.main }}
                   {{ $topLevel := replace .URL "/" "" }}
-                  <li class="dropdown{{ if not $current.IsHome }}{{ if or (eq $current.RelPermalink .URL) (eq $current.Type $topLevel) }} active{{ end }}{{ end }}">
+                  {{ $li_dropdown_klass := "dropdown" }}
```

Comment:
u might wanna use english terms here, i.e. `li_dropdown_class` (and not `*_klass`)

Reply:
`class`  in object oriented languages is a reserved word.

Reply:
for instace Java => https://en.wikipedia.org/wiki/List_of_Java_keywords#List

Reply:
doesn't seem to be case in Golang: https://golang.org/ref/spec#Keywords

Reply:
well I think you are agree with me about `this file it's a go template with html format`.

I would avoid to use the reserved words from: [go, html, javascript and css]. And `class` it's a reserved word from javascript in this case: http://www.javascripter.net/faq/reserved.htm

Reply:
Well, only `class` itself is reserved, not `*_class` or `class_*` etc. But in the end, people will probably figure out the meaning of `klass`... it just bothered me "visually" when I first saw it 😄

Reply:
finally I did as you proposal `li_dropdown_class`